app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def create
+    @user = Post.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(params.require(:comment).permit(:text))
+    @comment.author = current_user
+    @comment.post = @post
+
+    if @comment.save
+      redirect_to user_post_path(@user, @post)
+    else
+      render :new
+    end
+  end
+end
```

Comment:
- [ ] Please add `new` method as you added the `create` method to be able to create comments properly.
- [ ] Kindly also add a view for the `new` method in `views/comments/new.html.erb`